### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by XRichardroidX**
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
-  Implement popups Windows for both mobile and desktop screen sizes.
- use a JavaScript array to store all of the information for all projects.
- update the main page so that the projects section is created

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,7 +160,7 @@ <h4 class="abc">Multi-Post Stories Gain+Glory</h4>
                       <a href="#" class="skill-content">HTML</a>
                     </li>
                   </ul>
-                  <div class="button">See Projects</div>
+                  <div class="button" onclick="WindowPopUp('F1')">See Projects</div>
```
Comment (Reem-lab) [inline index.html]: - [ ] As I can see you didn't remove the works section code from your HTML file but according to the requirements I think you should remove all the hard coded section here and create it dynamically when the page loads so kindly work on that 💯🚀💪🏻🔥

image

File: script.js [hunk 2]
```diff
@@ -38,4 +38,152 @@ function OptionClick(){
...
+          'JavaScript', 
+          'HTML'
+         ],
+      "Sixth": [
+         'Assembly', 
+         "PHP",
+          'C', 
+          'C++'
+         ],
+   };
```
Comment (Reem-lab) [inline script.js]: - [ ] Great jon working on the design but I think here there is a note when I see the pop up I can see the heading above the two buttons but it in the Figma it should be there a space between them so kindly make a gap between the heading and the two buttons 


- Your Design
image

- Figma
image

File: README.md [hunk 3]
```diff
@@ -12,11 +12,8 @@
 
  ## Live Demo 
 
-<center><img title="Mobile" height="250" src="images/Mobile.png"></center>
-<center><img title="Desktop" height="250" src="images/Desktop.png"></center>
-![alt text](https://github.com/[username]/[reponame]/blob/[branch]/image.jpg?raw=true)
-[Live Demo Link](https://xrichardroidx.github.io/My-Microverse-Portfolio/)
- -->
+<img title="Mobile" height="250" src="images/Mobile.png">
+<img title="Desktop" height="250" src="images/Desktop.png">
```
Comment (Reem-lab) [inline README.md:16]: - [ OPTIONAL ] I recommend adding in your README the aothers authors who contribute with you in this project to show that you work with team so in the future recruiters can see that 🔥💪🏻🚀💯☺

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻 


Good job so far! 🚀💪🏻💯🔥🔥🔥🔥
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionalit✅
📌 Good commit messages ✅
📌 Professional README ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey Team @XRichardroidX, @Alabi12 :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following Requirements
:heavy_check_mark: Following Github Flow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**